tests/regression/patterns/llvm/rhel-82-83/bio_endio.ll

```diff
@@ -0,0 +1,113 @@
+; Found differences in functions called by bio_endio
+;
+; percpu_ref_put_many differs:
+;   Callstack (snapshots/linux-4.18.0-193.el8):
+;   bio_uninit at block/bio.c:1865
+;   bio_disassociate_blkg at block/bio.c:248
+;   blkg_put at block/bio.c:2060
+;   percpu_ref_put at ./include/linux/blk-cgroup.h:537
+;   percpu_ref_put_many at ./include/linux/percpu-refcount.h:301
+;
+;   Callstack (snapshots/linux-4.18.0-240.el8):
+;   bio_uninit at block/bio.c:1924
+;   bio_disassociate_blkg at block/bio.c:249
+;   blkg_put at block/bio.c:2119
+;   percpu_ref_put at ./include/linux/blk-cgroup.h:542
+;   percpu_ref_put_many at ./include/linux/percpu-refcount.h:325
+;
+;   Diff:
+;   *************** static inline void percpu_ref_put_many(struct percpu_ref *ref, unsigned long nr)
+;   *** 279,281 ***
+;
+;   ! 	rcu_read_lock_sched();
+;
+;   --- 303,305 ---
+;
+;   ! 	rcu_read_lock();
+;
+;   *************** static inline void percpu_ref_put_many(struct percpu_ref *ref, unsigned long nr)
+;   *** 286,288 ***
+;
+;   ! 	rcu_read_unlock_sched();
+;     }
+;   --- 310,312 ---
+;
+;   ! 	rcu_read_unlock();
+;     }
```

Comment:
It would help to see diff of the `bio_endio` function here to understand the pattern.

Reply:
This should be the full diff (all commented diffs should be complete). There is the option of copy-pasting the code as a whole, but I feel that might be a bit extreme. I think that simply explaining that the pattern is composed of two function calls with some code in between should be sufficient.

Reply:
Ah, I see now. The actual problem here is that the pattern sides are named `diffkemp.bio_endio.*` while the pattern is applied to `percpu_ref_put_many`. This is what confused me, b/c the other patterns are named by the function containing the actual difference, not by the originally compared one.

Reply:
Oh, you are correct. That's a mistake on my part. I'll change that.